### PR TITLE
Adding Courses for Hindi Language

### PR DESCRIPTION
**Pull request by Ravikisha**
## What does this PR do?
Add resource(s)

### Description
Adding online free courses for the Hindi language

### Why is this valuable (or not)?
Learners will learn some courses in Hindi and some good YouTubers will be recognized. 

### How do we know it's really free?
All the resources are from YouTube and free websites. These courses will be free always.

#### Courses
- Laravel
- Asp.Net
- React Native
- Game Development
- Spring Boot

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -10,11 +10,13 @@
 * [Blockchain](#blockchain)
 * [C](#c)
 * [C#](#csharp)
+    * [ASP.NET](#asp.net)
 * [C++](#cpp)
 * [Competitive Programming](#competitive-programming)
 * [Compiler Design](#compiler-design)
 * [Computer Graphics](#computer-graphics)
 * [Computer Organization and Architecture](#computer-organization-and-architecture)
+* [Cyber Security](#cyber-security)
```
Comment (eshellman) [inline courses/free-courses-hi.md:19]: Most of our lists just use "Security"

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -146,6 +151,14 @@
 * [C# Tutorial in Hindi for Beginners](https://www.youtube.com/playlist?list=PLV8vIYTIdSnYTYr0bmIbfzYii0YQSPocB) - Easy Engineering Classes
 
 
+### <a id="asp.net"></a>ASP.NET
+
+* [ASP .Net in Hindi](https://www.youtube.com/playlist?list=PLbsXhdgwIKL1g2vE86yGBK_RYuqt98yWL) - ComputerHindiNotes
+* [ASP.NET CORE 6.0 TUTORIALS IN HINDI](https://www.youtube.com/playlist?list=PL18HZjtdIA4DiYGQj1zst6myBAVE3wgMg) - Technology Keeda
```
Comment (eshellman) [inline courses/free-courses-hi.md:157]: no ALL CAPS

File: courses/free-courses-hi.md [hunk 3]
```diff
@@ -146,6 +151,14 @@
...
+* [ASP.NET CORE TUTORIALS IN HINDI](https://www.youtube.com/playlist?list=PL18HZjtdIA4Al-wYHC-i2TA-lgXLvHVmB) - Technology Keeda
```
Comment (eshellman) [inline courses/free-courses-hi.md:158]: no ALL CAPS

File: courses/free-courses-hi.md [hunk 4]
```diff
@@ -191,6 +204,15 @@
 * [Introduction to Computer Organization and Architecture](https://youtube.com/playlist?list=PLBlnK6fEyqRjC2nTHdeUtWFkoiPVespkc) - Neso Academy
 
 
+### Cyber Security
+
+* [cyber security course for beginners - Urdu/Hindi](https://www.youtube.com/playlist?list=PLKJfBg0XdWkevCEJ64RK11LylBNX2-zbk) - hashintelligence
+* [Cyber Security Tutorial In Hindi](https://www.youtube.com/playlist?list=PL-JvKqQx2AteIbm-z4X709scVr9OaHpIY) - University Academy
+* [Cyber Security Tutorials In HIndi](https://www.youtube.com/playlist?list=PL0fjgIGwLMWTFmZoLdEPoI9azA-osxcQj) - TechChip
+* [Ethical Hacking Full Couse || Cyber Security in Hindi](https://www.youtube.com/playlist?list=PLa2xctTiNSCibSUhgYI2RT_loUJP9rDN6) - Masters in IT
```
Comment (eshellman) [inline courses/free-courses-hi.md:212]: escape the `|`

File: courses/free-courses-hi.md [hunk 5]
```diff
@@ -315,7 +342,16 @@
 * [Java Tutorials for Beginners](https://www.youtube.com/playlist?list=PLlhM4lkb2sEhfuXL-2BDrJ67WkUdQ2v9b) - Deepak Panwar, Smart Programming
 * [Java Tutorials For Beginners In Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9agS67Uits0UnJyrYiXhDS6q) - CodeWithHarry
 * [Microservice Tutorial in Hindi 2023](https://www.youtube.com/playlist?list=PL0zysOflRCelb2Y4WOVckFC6B050BzV0D) - Learn Code With Durgesh
+* [Spring Framework Tutorial with SpringBoot | in Hindi| Complete Course | CoderX Ankit](https://www.youtube.com/playlist?list=PLiOIhBfKi8AIYNWEjYy1zZFe1N8GvPRHR) - CoderX Ankit
```
Comment (eshellman) [inline courses/free-courses-hi.md:345]: escape the `|`

File: courses/free-courses-hi.md [hunk 6]
```diff
@@ -459,6 +504,16 @@
 * [PHP Tutorials in Hindi](https://youtube.com/playlist?list=PLu0W_9lII9aikXkRE0WxDt1vozo3hnmtR) - CodeWithHarry
 
 
+### Laravel
+
+* [Laravel (Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhijEqjCa63l0GkWh5EsG5iTR) - Geeky Shows
+* [Laravel 10 Tutorial Course in Hindi / Urdu](PL0b6OzIxLPbz7JK_YYrRJ1KxlGG4diZHJ) - Yahoo Baba
```
Comment (eshellman) [inline courses/free-courses-hi.md:510]: you forgot the link

File: courses/free-courses-hi.md (hunk 2 above)
Comment (eshellman) [reply courses/free-courses-hi.md:157]: shouldn't this one supercede the following one? Why both?

**Review by eshellman — CHANGES_REQUESTED** [review]
It's easier to get a PR approved if it makes fewer changes

**Comment by Ravikisha** [comment]
**Update Requested Changes 👍**

Thank you for addressing the feedback provided in the initial review. I've reviewed the changes you made.

Feel free to reach out if you have any questions or need further clarification on any of these points. Let's work together to ensure the code meets your standards and requirements. Thanks for your effort!

**Comment by Ravikisha** [comment]
I resolved the conflict and Thanks for your effort.

File: courses/free-courses-hi.md [hunk 7]
```diff
@@ -373,7 +398,16 @@
...
-* [ReactNative Tutorials for Begginers](https://youtube.com/playlist?list=PL8kfZyp--gEXs4YsSLtB3KqDtdOFHMjWZ) - Programming with Mash
+
+
+### React Native
+
+* [React Native hindi tutorial](https://www.youtube.com/playlist?list=PL8p2I9GklV479IV5cXwKXqGOcCOu0bPXW) - Code Step By Step
+* [React Native Tutorial in Hindi](https://www.youtube.com/playlist?list=PL8p2I9GklV468O2wk-n8Q1KmtMhnHHj4C) - Code Step By Step
+* [React Native Tutorial In Hindi - 2023](https://www.youtube.com/playlist?list=PLvN7nvnjkvpQribRyQ4r0FYZxKsPLGciy) - Code Diggers
+* [React Native Tutorial in Hindi 2022](https://www.youtube.com/playlist?list=PLwGdqUZWnOp354xMD8u0hxX-1qmCvfLiY) - Thapa Technical
+* [React Native Tutorials (mobile app development) in hindi](https://www.youtube.com/playlist?list=PLB97yPrFwo5gxB5SuNWzH73t2Su65KN2f) - CODERS NEVER QUIT
```
Comment (eshellman) [inline courses/free-courses-hi.md:409]: no ALL CAPS

**Comment by Ravikisha** [comment]
Sorry for the issue.

File: courses/free-courses-hi.md (hunk 5 above)
Comment (eshellman) [reply courses/free-courses-hi.md:338]:
```suggestion
* [Spring Framework Tutorial with SpringBoot  in Hindi Complete Course](https://www.youtube.com/playlist?list=PLiOIhBfKi8AIYNWEjYy1zZFe1N8GvPRHR) - CoderX Ankit
```

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books! For future PRs, you can minimize conflicts by working on one section at a time.

**Pull request merged**